### ISSUE TITLE
When selecting "Join with Zoom" button, nothing happens.

### ISSUE THREAD

**Issue by Henk321** [code4recovery/12-step-meeting-list#172]
I have had several users complain that when they press the "Join with Zoom" button from the meeting description (for example: https://cowichanaa.ca/?tsml_meeting=brown-baggers) nothing happens. At first I thought the zoom link was malformed but discovered that it does work from the meeting guide app. The same link also works from the page on which we list all our on-line meetings. 

I tried the button myself and found it not to work. I then accessed the meeting through our on-line meeting page. I am using Chrome and Windows 10.  I tried right clicking the button and using open link in a new tab and a new tab opened and it loaded the meeting description page all over again. No zoom dialogue appeared anywhere. 

Not sure what to do other than report it here and hope there is a solution.

Thank you.

Henk

**Comment by Henk321** [comment]
I just tried the same thing on my android phone, using Chrome to visit the website. Button did not work.
Back to my computer, tried Firefox. Nothing happened.
Firefox, on my phone, nothing.

**Comment by tech2serve** [comment]
@Henk321, 

How do you have the Permalinks set? Please attach a screen cap of those settings. 

As an example:

<img width="1203" alt="image" src="https://user-images.githubusercontent.com/31904316/94623745-870d6600-027a-11eb-87d1-c38e0915dfe1.png">

**Comment by Henk321** [comment]
Our permalink settings:

[image: image.png]

**Comment by Henk321** [comment]
Our Permalinks settings:

![image](https://user-images.githubusercontent.com/29762717/94626298-a226a880-026f-11eb-9662-ea1583c23453.png)

**Comment by tech2serve** [comment]
Tbh, I'm guessing here because your situation, afaik, is isolated. In the couple of situations where I have seen this, I have not been able to reproduce the problem when I visited their site. Hence, I usually suggest users clear their browser cache.

However, I was able to see the incorrect behavior when I visit your site. I wondered about your permalink settings. But, when I changed mine on my dev site to match yours, I was still able open a Zoom meeting. 

Typically when we are able to reproduce issues on the website having the problem, but not on our own dev sites, we need to look more closely into your website.

1. Please ensure you're using the most recent version of the TSML plugin.
1. What version of WordPress are you using?
1. What version of PHP?
1. If you're able to reset the cache for your website (through your hosting provider interface), does doing so make any difference?

**Comment by Henk321** [comment]
Thank you for getting back to us so quickly.

TSML plugin version: 3.8.0
WordPress version: 5.5.1
PHP version: 7.3

Reset the cache?  I used the flush all button from:

![image](https://user-images.githubusercontent.com/29762717/94634960-5b907880-0286-11eb-94c8-79699c8d87ec.png)


After doing so, I was able to join a meeting from the TSML plug from both my laptop and my phone!! Nice it's working again. Thank you, I would never have thought to do so.

How often should I flush the cache? It's not a maintenance task I have done before but can set up a reminder to myself to do so once a week or once a month. 

Thank you for taking time to help us. Lots of gratitude here!! :)
